File: lista1/zadanie1_java/app/src/main/java/com/example/zadanie1_java/MainActivity.java
package com.example.zadanie1_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private int r1 = 0;
    private int r2 = 0;
    private int score = 0;
    private final Random random = new Random();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        roll();
    }

    private void roll() {
        ((TextView) findViewById(R.id.points)).setText("Score: " + score);
        r1 = random.nextInt(10);
        r2 = random.nextInt(10);
        ((Button) findViewById(R.id.leftClick)).setText("" + r1);
        ((Button) findViewById(R.id.rightClick)).setText("" + r2);
    }

    public void buttonLeft(View view) {
        if (r1 >= r2) {
            score++;
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show();
        } else {
            score--;
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }

    public void buttonRight(View view) {
        if (r1 <= r2) {
            score++;
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show();
        } else {
            score--;
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }
}

File: lista1/zadanie1_kotlin/app/src/main/java/com/example/zadanie1_kotlin/MainActivity.kt
package com.example.zadanie1_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import kotlin.random.Random

class MainActivity : AppCompatActivity() {
    private var score = 0
    private var r1 = 0
    private var r2 = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        roll()
    }

    fun roll(){
        findViewById<TextView>(R.id.points).text = "Punkty: " + score
        r1 = Random.nextInt(10)
        r2 = Random.nextInt(10)
        findViewById<TextView>(R.id.leftClick).text = r1.toString()
        findViewById<TextView>(R.id.rightClick).text = r2.toString()

    }

    fun buttonRight(view: View) {
        if (r2 >= r1){
            score++
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show()
        }
        else{
            score--
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }

    fun buttonLeft(view: View) {
        if (r1 >= r2){
            score++
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show()
        }
        else{
            score--
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }
}